tests/test_modeling_common.py

```diff
@@ -419,7 +405,6 @@ def test_head_pruning_integration(self):
             if "head_mask" in inputs_dict:
                 del inputs_dict["head_mask"]
 
-            config.output_attentions = True
```

Comment:
In this test we should make sure that in the  forward function `output_attentions=True` . We can just add `inputs_dict["output_attentions"] = True` to the input.